src/app/faq/faq.component.sass

```diff
@@ -0,0 +1,41 @@
+@import './src/components/variables.sass'
+
+.faq
```

Comment:
utilizing proper SASS going forward will help with readability. Check some other SASS files that i had originally commit. 

an example would be
```
.faq
  display: grid
  grid-gap: 20px
  
  & p
    font-size: 40px
    line-eight: 2.75rem
```